src/main/java/vinyldns/java/model/batch/BatchResponse.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2018 Comcast Cable Communications Management, LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package vinyldns.java.model.batch;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.StringJoiner;
+
+public class BatchResponse {
+  private String id;
+  private String userId;
+  private String userName;
+  private String comments;
+  private Date createdTimestamp;
+  private List<SingleChange> changes;
```

Comment:
`private BatchChangeStatus status` is missing (as well as the enumeration class for `BatchChangeStatus`). Values for `BatchChangeStatus` are: `pending`, `complete`, `failed`, and `partialFailure` (see: https://www.vinyldns.io/api/batchchange-model.html#batch-change-attributes-).

Reply:
done